rundevtests.sh

```diff
@@ -4,20 +4,31 @@
 try() {
     "$@" || return 1
 }
-export K6_TEST_FILE=sc1-isis-proposal-submission-test
+export K6_TEST_FILE=sc1-fap-review-view-proposal-test
 export XK6_BROWSER_LOG="fatal"
 export K6_BROWSER_LOG="error"
 export ENVIRONMENT="local"
 export BROWSER_BASE_URL=http://duo-reverse-proxy:80
 export GRAPHQL_URL=http://duo-reverse-proxy:80/graphql
 export PROPOSAL_LOOKUP_URL=http://localhost:4006/ProposalWebService/ProposalLookupWebService?wsdl
 export SETUP_TOTAL_USERS=50
-export USER_STARTING_ID=-240800000
-export TEST_SETUP_CALL_ID=54
-export K6_PS_ITERATIONS=2
-export INSTRUMENT_ID=6
+export USER_STARTING_ID=-220800000
+export SETUP_TOTAL_REVIEWERS=50
+export REVIEWER_STARTING_IDS=-220800000
+export TEST_SETUP_CALL_ID=145
+export K6_PS_ITERATIONS=1
+export K6_PS_VUS=50
+export TEST_SETUP_FAP_ID=1
+export TEST_SET_UP_PROPOSAL_PKS=1
+export TEST_SET_UP_INSTRUMENT_ID=37
```

Comment:
We may still need these `export TEST_SETUP_CALL_ID=54
export K6_PS_ITERATIONS=2
export INSTRUMENT_ID=6`